package.json

```
@@ -15,8 +15,7 @@
   "scripts": {
     "test:unit": "vitest --run",
     "test-watch:unit": "vitest --watch",
-    "test:storybook": "test-storybook --testTimeout=60000",
-    "test-watch:storybook": "test-storybook --watch --testTimeout=60000",
+    "test-watch:unit:ui": "vitest --ui",
```

デバッグに便利だったので足しました。

---

こっちって`--watch`は不要な感じでしょうか 👀

---

他の例調べたら`test:ui`が多そうな感じでした。
多分引数名を後ろにつける感じっぽい。`--coverage`だったら`:coverage`みたいな。
https://github.com/search?q=vitest+--ui&type=code&p=2

ということで合わせるならこんな感じですかね！
```suggestion
    "test-ui:unit": "vitest --ui",
```
（ちょっとなんかいびつなのでいつか整理したいですね。。。特に後ろに`unit`をつけるのが本当に正しいのかどうかが微妙になってきた！）
（プロジェクトが成長してコンポーネントテストとかで書けるようになったので）